data/yaml/joint.yaml

```diff
@@ -1287,6 +1287,9 @@ storynlp:
 stylevar:
 - W17-49
 - W18-16
+syntaxfest:
+  2021: [quasy, depling, tlt, udw]
```

Comment:
These have to be volume identifiers, e.g., `2021.quasy-1`, etc

Reply:
After its built, they should all show up here (currently empty): https://preview.aclanthology.org/syntaxfest-ingestion/venues/syntaxfest/

Reply:
yep